### PR TITLE
ci(SW-27140): Tests are executed in dependence on each other

### PR DESCRIPTION
**Pull request by DennisGarding**

### CODE REVIEW

**Comment by aragon999** [comment]
Just a minor remark, not related to the content of this PR, I guess I would name the commit something like `ci(SW-27140): Execute tests in dependence of each other` :) 

The reasoning behind this tag would be, that `ci` commits can be omitted from changelogs, as they are not really relevant for downstream :-)

File: composer.lock [hunk 1]
```diff
@@ -2405,39 +2405,36 @@
         },
         {
             "name": "laminas/laminas-code",
-            "version": "4.7.1",
+            "version": "4.11.0",
```
Comment (mitelg) [inline composer.lock]: please revert this. otherwise PHP7.4 can no longer used

Comment (mitelg) [reply composer.lock]: this will probably cause a fail in the pipeline once a PHP7.4 job is executed. at least I hope so :grimacing:

**Review by mitelg — CHANGES_REQUESTED** [review]

**Review by ennasus4sun — APPROVED** [review]

File: composer.lock (hunk 1 above)
Comment (DennisGarding) [reply composer.lock]: ohh no.. i forgot to revert it. Thnaks

**Comment by mitelg** [comment]
looks good so far, but you need a rebase on "next" branch and then resolving the conflicts

**Comment by aragon999** [comment]
And the scope of the commit (i.e. the ticket number) should be before the colon :-) 
https://www.conventionalcommits.org/en/v1.0.0/#summary

Not sure why danger did not ran.

**Review by mitelg — CHANGES_REQUESTED** [review]
needs a rebase due to conflicts

**Review by mitelg — APPROVED** [review]

**Pull request merged**